kentik_api_library/kentik_api/requests_payload/manual_mitigations_payload.py

```diff
@@ -16,3 +21,111 @@ def from_json(cls, json_string):
 
     def status(self) -> str:
         return self.result
+
+
+@dataclass
+class _Alarm:
+    alarm_id: int
+    row_type: str
+    alarm_state: str
+    alert_id: int
+    mitigation_id: Optional[int]
+    threshold_id: int
+    alert_key: str
+    alert_dimension: str
+    alert_metric: List[str]
+    alert_value: float
+    alert_value2nd: float
+    alert_value3rd: float
+    alert_match_count: int
+    alert_baseline: int
+    alert_severity: str
+    baseline_used: int
+    learning_mode: int
+    debug_mode: int
+    alarm_start: str
+    alarm_end: str
+    alarm_last_comment: Optional[str]
+    mit_alert_id: int
+    mit_alert_ip: str
+    mit_threshold_id: int
+    args: str
+    id: int
+    policy_id: int
+    policy_name: str
+    alert_key_lookup: str
+
+    def to_alarm(self) -> Alarm:
+        dic = self.__dict__
+        try:
+            dic["alarm_end"] = datetime.strptime(self.alarm_end, "%Y-%m-%d %H:%M:%S")
```

Comment:
is this field optional so it is placed in try..except?

Reply:
This field is always passed, but it can contain string which cannot be parsed as datetime (year, month and day are set to 0 which are invalid values for datetime)